File: javascript/10_sum_primes.js

function isPrime(numb){
  for(var i=2; i<=Math.floor(Math.sqrt(numb)); i++){
    if (numb % i == 0){
      return false;
    }
  }
  return true;
}

var count = 10;
for(var p = 7; p < 2000000; p += 6){
  if (isPrime(p)) count += p;
  if (isPrime(p+4)) count += (p+4);
}
  
console.log(count);

File: python/10_sum_primes.py
import math

def isPrime(numb):
  for i in xrange(2,int(math.sqrt(numb))+1):
    if (numb % i == 0):
      return False
  return True


count = 10
for p in xrange(7,2000000,6):
  if isPrime(p): count += p
  if isPrime(p+4): count += (p+4)

print count